administrator/components/com_fields/src/Plugin/FieldsListPlugin.php

```diff
@@ -70,8 +70,15 @@ public function getOptionsFromField($field)
 
 		// Fetch the options from the plugin
 		$params = clone $this->params;
+		
+		foreach ($params->get('options', array()) as $option)
+		{
+			$op = (object) $option;
+			$data[$op->value] = $op->name;
+		}
+		
```

Comment:
```suggestion

		foreach ($params->get('options', array()) as $option)
		{
			$op = (object) $option;
			$data[$op->value] = $op->name;
		}

```
Empty lines should not contain spaces or tabs.

Reply:
Same for line 81 below.

Reply:
Yes, sir, I've made changes according to your suggestion.